docs/toolbox/accessibility_indicators/catchments.md

```diff
@@ -231,6 +231,84 @@ Per default, the catchment areas are calculated in polygon shape. In case you wa
 </Tabs>
 
 
+  </TabItem>
+
+  <TabItem value="car" label="Car" className="tabItemBox">
+
+#### Car
+
+In this routing mode, users can compute accessibility for the car street network.
+
+:::tip Hint
+
+For further insights into the Routing algorithm, visit [Car](../../routing/car).
+
+:::
+
+<div class="step">
+  <div class="step-number">4</div>
+  <div class="content">Pick if you like to calculate the catchment area based on <b>time</b> or <b>distance</b>.</div>
+</div>
+
+<Tabs>
+  <TabItem value="time" label="Time" default className="tabItemBox">
+
+#### Time
+
+<div class="step">
+  <div class="step-number">5</div>
+  <div class="content">Set the configurations for <code>Travel time limit</code>, <code>Travel speed</code>, and <code> Number of breaks</code>.</div>
+</div>
+
+<img src={require('/img/toolbox/accessibility_indicators/catchments/walk_config_time.png').default} alt="walking-time configurations" style={{ maxHeight: "300px", maxWidth: "300px"}}/>
+
+:::tip Hint
+
+For defining which travel time limits are suitable for which amenity, the ["Standort-Werkzeug"](https://www.chemnitz.de/chemnitz/media/unsere-stadt/verkehr/verkehrsplanung/vep2040_standortwerkzeug.pdf) of the City of Chemnitz can provide helpful guidance.
```

Comment:
I think the "Standort-Werkzeug" recommendations might be intended more for walking/cycling, so we can omit this for car.